content/ch-algorithms/quantum-phase-estimation.ipynb

```diff
@@ -115,7 +115,7 @@
     "\\end{aligned}\n",
     "where $k$ denotes the integer representation of n-bit binary numbers. \n",
     "\n",
-    "iv. **Inverse Fourier Transform**: Notice that the above expression is exactly the result of applying a quantum Fourier transform as we derived in the notebook on [Quantum Fourier Transform and its Qiskit Implementation](qft.ipynb). Recall that QFT maps an n-qubit input state $\\vert x\\rangle$ into an output as\n",
+    "iv. **Inverse Fourier Transform**: Notice that the above expression is exactly the result of applying a quantum Fourier transform as we derived in the notebook on [Quantum Fourier Transform and its Qiskit Implementation](quantum-fourier-transform.ipynb). Recall that QFT maps an n-qubit input state $\\vert x\\rangle$ into an output as\n",
```

Comment:
Thanks for this, we generally link to the 
```suggestion
    "iv. **Inverse Fourier Transform**: Notice that the above expression is exactly the result of applying a quantum Fourier transform as we derived in the notebook on [Quantum Fourier Transform and its Qiskit Implementation](https://qiskit.org/textbook/ch-algorithms/quantum-fourier-transform.html). Recall that QFT maps an n-qubit input state $\\vert x\\rangle$ into an output as\n",
```
Thanks for doing this, we generally link to the live site so the links work both as local notebooks and also on qiskit.org/textbook. Not a perfect solution but seems to work fine for now.

Reply:
Yes. Thanks for clarifying that. I previously contributed to qiskit documentation and there only local notebook path are needed so I thought. But from now I will try to add site links only.